shotover-proxy/src/protocols/cassandra_protocol2.rs

```diff
@@ -913,46 +914,4 @@ mod cassandra_protocol_tests {
         }];
         test_frame_codec_roundtrip(&mut codec, &bytes, messages);
     }
-
-    #[test]
-    fn test_process_cassandra_frame() {
```

Comment:
Did this test get moved somewhere else? Or do we have a good reason for removing it?

Reply:
I thought it was superceded by the test_codec_* tests but I noticed that none of them test Opcode::Options, so I added a test case for that.